handlers/main.yml

```diff
@@ -36,6 +36,11 @@
       src: /dev/shm
       state: remounted
 
+- name: Purge_packages
+  ansible.builtin.apt:
+      purge: true
```

Comment:
I'm not sure this will achieve what you're hoping. I thought you can only purge apps on an individual basis so you'll need to pass it the package name and `state: absent`

Reply:
hi @zac90 

Fully tested now and moved to usinf purge: {{ variable }}now. This does work on the paclage module as that is turn references the systems package manager. So great fix and thanks again for the feedback

uk-bolly